test/parallel/test-v8-deserialize-buffer.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
Do we need a `Flags: --pending-deprecations`?

Reply:
I'll add one, gimme a sec.

Reply:
Okay, most other tests add `--no-warnings` too, but doesn't that kinda defeat the purpose of the test?

Reply:
Example:

```js
// Flags: --pending-deprecation --no-warnings
'use strict';

const common = require('../common');

process.on('warning', common.mustNotCall());

Buffer.of(0, 1);

```

Reply:
Looks like it doesn't, the test fails with no warnings getting printed out, exactly like it should. Adding the line.